device.py

```
@@ -0,0 +1,243 @@
+    if board_name == board_vendor or board_vendor == "":
+        output_file.write(f"CONFIG_TT_BOARD_NAME=\"{board_name}\"\n")
+    else:
+        output_file.write(f"CONFIG_TT_BOARD_NAME=\"{board_vendor} {board_name}\"\n")
+    output_file.write(f"CONFIG_TT_BOARD_ID=\"{device_id}\"\n")
+
+def write_core_variables(output_file, device_properties: ConfigParser):
+    idf_target = get_property_or_exit(device_properties, "hardware", "target")
+    output_file.write("# Target\n")
+    output_file.write(f"CONFIG_IDF_TARGET=\"{idf_target.lower()}\"\n")
+    output_file.write("# CPU\n")
+    output_file.write(f"CONFIG_ESP_DEFAULT_CPU_FREQ_MHZ_240=y\n")
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)